.gitignore

```diff
@@ -114,4 +114,5 @@ env/
 
 ### Configuration files ###
 .env
+.env.test
```

Comment:
No, this should not be in .gitignore - remove this line. It should be necessary for the unit/integration tests to run and it should not contain any secrets, so why ignore it?

Fortunately just adding a file to .gitignore when it's already being tracked won't remove it. So .env.test is still in the repo https://github.com/sujitn/DevOps-Course-Exercise/blob/module7/.env.test